apps/site/pages/en/learn/typescript/run.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Running TypeScript with a runner
+layout: learn
+authors: AugustinMauroy
+---
+
+# Running TypeScript with a runner
+
+In the previous article, we learned how to run TypeScript code using transpilation. In this article, we will learn how to run TypeScript code using a runner.
+
+## Running TypeScript code with `ts-node`
+
+[ts-node](https://typestrong.org/ts-node/) is a TypeScript execution environment for node.js. It allows you to run TypeScript code directly in node.js without the need to compile it first. But it's not typechecking your code. So we recommend to type check your code first with `tsc` and then run it with `ts-node` before shipping it.
+
+To use `ts-node`, you need to install it first:
+
+```bash
+npm i -D ts-node
+```
+
+Then you can run your TypeScript code like this:
+
+```bash
+npx ts-node example.ts
+```
+
+## Running TypeScript code with `tsx`
+
+[tsx](https://tsx.is/) is another TypeScript execution environment for node.js. It allows you to run TypeScript code directly in node.js without the need to compile it first. But it's not typechecking your code. So we recommend to type check your code first with `tsc` and then run it with `tsx` before shipping it.
+
+To use `tsx`, you need to install it first:
+
+```bash
+npm i -D tsx
+```
+
+Then you can run your TypeScript code like this:
+
+```bash
+npx tsx example.ts
+```
+
+### Registering `tsx` via `node`
+
```

Comment:
We need a paragraph about what mean "registering" someone on loader team can write it 😁😁😁